app/controllers/case_logs_controller.rb

```diff
@@ -76,6 +76,11 @@ def destroy
     end
   end
 
+  def filter
+    cookies[:case_logs_filters] = { status: params[:status] }.to_json
```

Comment:
I think we should use `session[:case_logs_filters]` rather than `cookies[:case_logs_filters]`. I wouldn't expect filter choices to persist beyond your current session?